src/dotnet/commands/dotnet-new/Program.cs

```diff
@@ -15,98 +15,46 @@ namespace Microsoft.DotNet.Tools.New
-                            where resourceName.Contains(templateDir)
-                            select resourceName;
 
-            var resourceNameToFileName = new Dictionary<string, string>();
-            bool hasFilesToOverride = false;
-            foreach (string resourceName in resources)
+            using (var resource = thisAssembly.GetManifestResourceStream($"dotnet.commands.dotnet_new.{templateName}.zip"))
             {
-                string fileName = GetFileNameFromResourceName(resourceName);
+                var archive = new ZipArchive(resource);
 
-                using (var resource = thisAssembly.GetManifestResourceStream(resourceName))
+                try
                 {
-                    var archive = new ZipArchive(resource);
-
-                    try
+                    // Check if other files from the template exists already, before extraction
+                    IEnumerable<string> fileNames = archive.Entries.Select(e => e.FullName);
+                    foreach (var entry in fileNames)
                     {
-                        // Check if other files from the template exists already, before extraction
-                        IEnumerable<string> fileNames = archive.Entries.Select(e => e.FullName);
-                        foreach (var entry in fileNames)
+                        if (File.Exists(Path.Combine(Directory.GetCurrentDirectory(), entry)))
                         {
-                            if (File.Exists(Path.Combine(Directory.GetCurrentDirectory(), entry)))
-                            {
-                                Reporter.Error.WriteLine(string.Format(LocalizableStrings.ProjectContainsError, languageName, entry));
-                                return 1;
-                            }
+                            Reporter.Error.WriteLine(string.Format(LocalizableStrings.ProjectContainsError, languageName, entry));
+                            return 1;
                         }
+                    }
 
-                        string projectDirectory = Directory.GetCurrentDirectory();
-
-                        archive.ExtractToDirectory(projectDirectory);
+                    string projectDirectory = Directory.GetCurrentDirectory();
 
-                        ReplaceFileTemplateNames(projectDirectory);
+                    archive.ExtractToDirectory(projectDirectory);
 
-                        if (!isMsBuild)
-                        {
-                            ReplaceProjectJsonTemplateValues(projectDirectory);
-                        }
-                    }
-                    catch (IOException ex)
-                    {
-                        Reporter.Error.WriteLine(ex.Message);
-                        hasFilesToOverride = true;
-                    }
+                    ReplaceFileTemplateNames(projectDirectory);
+                }
+                catch (IOException ex)
+                {
+                    Reporter.Error.WriteLine(ex.Message);
```

Comment:
Since my change is only to allow 2 template names that contain that same strings ("Console" and "Console1.1"), I'll let @krwq decided in his change whether this is the correct treatment for error handling or not.